signer/fourbyte/4byte.json

```diff
@@ -31711,7 +31711,7 @@
 "1e5296ef": "getMySellOrdersBlackMarketComplete()",
 "1e52aa52": "startBetRed()",
 "1e52ca35": "ROLE_SUPER()",
-"1e52f7b5": "accoutToPayLevel(address)",
+"1e52f7b5": "accountToPayLevel(address)",
```

Comment:
There are four typos in this file. These seem to be external symbols and I'm not sure I should change it.

Should I revert changes to this file?

Reply:
Fixed 👍